sdk/search/search-documents/src/odata.ts

```diff
@@ -31,7 +40,11 @@ export function odata(strings: TemplateStringsArray, ...values: unknown[]): stri
   for (let i = 0; i < strings.length; i++) {
     results.push(strings[i]);
     if (i < values.length) {
-      results.push(escapeQuotesIfString(values[i], strings[i]));
+      if (values[i] == null || values[i] == undefined) {
+        results.push(formatNullAndUndefined(values[i]));
```

Comment:
actually, since you do the if check inside of `formatNullAndUndefined` should we just do something like

```ts
const value = formatNullAndUndefined(values[i]);
results.push(escapeQuotesIfString(value, strings[i]));
```

and avoid doing if/else inside this for loop?